pyproject.toml

```diff
@@ -9,14 +9,27 @@ build-backend = "setuptools.build_meta"
 legacy_tox_ini = """
 [tox]
 
-[testenv]
+[testenv:tests]
 changedir = tests
 deps =
     coverage[toml]
 
 commands =
     coverage run -m unittest discover -p "*.py"
     coverage xml
+
+[testenv:examples]
+changedir = examples
+allowlist_externals = /bin/bash
+
+deps =
+    jupyter
+    jupyter_contrib_nbextensions
+    matplotlib
+    tqdm
+
+commands =
+    bash -c "for f in *.ipynb; do jupyter nbconvert --execute "$f" --to python; done"
```

Comment:
```suggestion
    bash -c "for file in *.ipynb; do jupyter nbconvert --execute "$file" --to python; done"
```

Reply:
Also, executing tests locally creates new python files in `examples`. Could we put these files in `.tox/examples` instead?

Reply:
done